src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,18 @@
 import React from 'react';
+import { Movie } from '../../types/Movie';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => (
+  <div className="movies">
+    {movies.map((movie) => {
+      return (
+        <MovieCard movie={movie} key={movie.imdbId} />
+      );
+    })}
```

Comment:
you can just use short syntax
```suggestion
    {movies.map((movie) => (
        <MovieCard movie={movie} key={movie.imdbId} />
      );
    )}
```